x-pack/plugins/maps/public/shared/layers/sources/wms_source.js

```diff
@@ -67,7 +77,6 @@ export class WMSSource extends AbstractTMSSource {
   }
 
   getUrlTemplate() {
-    console.warn('should compose url using url formatter, not string formatting');
     const styles = this._descriptor.styles ? this._descriptor.styles : '';
```

Comment:
maybe this is less verbose?

```suggestion
    const styles = this._descriptor.styles || '';
```